neuralop/layers/coda_blocks.py

```diff
@@ -0,0 +1,380 @@
+from functools import partial
+import logging
+import numpy as np
+import torch
+import torch
+import math
+from torch import nn
+import torch.nn.functional as F
+from .fno_block import FNOBlocks
+from .spectral_convolution import SpectralConv
+
+einsum_symbols = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+# Codomain Attention Blocks
+
+
+class CODABlocks(nn.Module):
+    """ Class for the Co-domain Attention Blocks (CODABlocks)
```

Comment:
Our docs use NumpyDoc, can you reformat this into a [Numpy docstring](https://numpydoc.readthedocs.io/en/latest/format.html)?

Reply:
Reformated.